lib/utils.js

```diff
@@ -2,14 +2,17 @@ var Promise = require('bluebird');
 var deepCopy = require('deep-copy');
 var traverse = require('traverse');
 
-module.exports.clone = function(obj, preserveFunctions){
+module.exports.clone = function(obj, circular){
```

Comment:
This function's prototype has changed, but I cannot see any references to it that has changed accordingly.